nixos/modules/services/backup/restic.nix

```
@@ -358,7 +358,7 @@ in
                 ${resticCmd} cat config > /dev/null || ${resticCmd} init
               ''}
               ${optionalString (backup.paths != null && backup.paths != []) ''
-                cat ${pkgs.writeText "staticPaths" (concatStringsSep "\n" backup.paths)} >> ${filesFromTmpFile}
+                cat ${pkgs.writeText "staticPaths" ((concatStringsSep "\n" backup.paths) + "\n")} >> ${filesFromTmpFile}
```

`lib.concatLines` does what we need out of the box:

https://github.com/NixOS/nixpkgs/blob/c68bae316281af1b80a1d12c6f56d2b92b6f2d3a/lib/strings.nix#L272-L298

```suggestion
                cat ${pkgs.writeText "staticPaths" (concatLines backup.paths)} >> ${filesFromTmpFile}
```